_layouts/tag.html

```diff
@@ -26,6 +26,8 @@ <h3 id="#{{ tag }}">#{{ tag }}</h3>
           </div>
         </div>
       </div>
+
+      <h3><a href="{{site.baseurl}}/tags/">See other tags</a></h3>
```

Comment:
Let's make this an `h4` instead to align it with the rest of the theme 😄

Reply:
You're right!